package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "build:demos": "node ./scripts/run.js prepublish demos",
-    "checkdeps": "node ./scripts/dependencies.test.js",
+    "checkdeps": "node ./scripts/check.dependencies.js --auto-fix",
```

Comment:
But as it does not do an install after checkdeps, that has no influence on the test which should fail anyway due to missing installed dependencies. The idea is that people run `npm run test` before the stage stuff and commit. With this workflow:

1. write code in new branch
2. test
3. all good =>git add ..., git cz